qemu/tests/cfg/balloon_stress.cfg

```diff
@@ -1,26 +1,32 @@
 - balloon_stress:
     type = balloon_stress
     virt_test_type = qemu
-    only Windows
     kill_vm = yes
     repeat_times = 500
```

Comment:
The case is "10000 times memory enlarge and evict". Should we update the polarion case to "500 times"?